pillar/heroku/ocw-studio.sls

```diff
@@ -190,7 +190,7 @@ heroku:
     THREEPLAY_API_KEY: __vault__::secret-operations/global/{{ business_unit }}/ocw-studio/threeplay_api_key>data>value
     THREEPLAY_CALLBACK_KEY: __vault__:gen_if_missing:secret-operations/global/{{ business_unit }}/ocw-studio/threeplay_callback_key>data>value
     USE_X_FORWARDED_PORT: True
-    VIDEO_S3_TRANSCODE_ENDPOINT: __vault__::secret-operations/global/{{ business_unit }}/ocw-studio/{{ environment }}/video_s3_transcode_endpoint>data>value
+    VIDEO_S3_TRANSCODE_ENDPOINT: __vault__::secret-ocw-studio/video_s3_transcode_endpoint
```

Comment:
Sorry, I didn't include all of the needed detail. It still needs the `>data>data>value` to get at the setting